src/components/Calculator.js

```diff
@@ -1,84 +1,89 @@
           </div>
           <div className="button-row">
-            <button type="button" className="button-column">
+            <button type="button" className="button-column" onClick={this.handleClick}>
               7
             </button>
-            <button type="button" className="button-column">
+            <button type="button" className="button-column" onClick={this.handleClick}>
               8
             </button>
-            <button type="button" className="button-column">
+            <button type="button" className="button-column" onClick={this.handleClick}>
               9
             </button>
-            <button type="button" className="button-column orange">
+            <button type="button" className="button-column orange" onClick={this.handleClick}>
               x
             </button>
           </div>
           <div className="button-row">
-            <button type="button" className="button-column">
+            <button type="button" className="button-column" onClick={this.handleClick}>
               4
             </button>
-            <button type="button" className="button-column">
+            <button type="button" className="button-column" onClick={this.handleClick}>
               5
             </button>
-            <button type="button" className="button-column">
+            <button type="button" className="button-column" onClick={this.handleClick}>
               6
             </button>
-            <button type="button" className="button-column orange">
+            <button type="button" className="button-column orange" onClick={this.handleClick}>
               -
             </button>
           </div>
           <div className="button-row">
-            <button type="button" className="button-column">
+            <button type="button" className="button-column" onClick={this.handleClick}>
               1
             </button>
-            <button type="button" className="button-column">
+            <button type="button" className="button-column" onClick={this.handleClick}>
               2
             </button>
-            <button type="button" className="button-column">
+            <button type="button" className="button-column" onClick={this.handleClick}>
               3
             </button>
-            <button type="button" className="button-column orange">
+            <button type="button" className="button-column orange" onClick={this.handleClick}>
               +
             </button>
           </div>
           <div className="button-row">
-            <button type="button" className="button-column-1">
+            <button type="button" className="button-column-1" onClick={this.handleClick}>
               0
             </button>
 
-            <button type="button" className="button-column orange">
+            <button type="button" className="button-column orange" onClick={this.handleClick}>
               =
             </button>
           </div>
```

Comment:
- [ ] Nice work so far 🔥. However, you're missing the `.` button implementation. Kindly add `.` to your calculation and implement its functionality too.

Your Implementation | Supposed Implementation
:---:|:---:
![](https://i.imgur.com/gRm22E0.png) | ![](https://github.com/microverseinc/curriculum-react-redux/raw/main/math-magicians/images/calculator.png)